<?xml version="1.0"?>
<launch>
	<node pkg="telepresence_activity" name="telepresence_activity_client" type="telepresence_activity_client" output="screen"/>
</launch>
